pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -761,6 +761,56 @@ func handleAppFlowMonitorDelete(ctxArg interface{}, key string,
 	log.Infof("handleAppFlowMonitorDelete(%s) done\n", key)
 }
 
+func handleAppVifIPTrigModify(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	log.Infof("handleAppVifIPTrigModify(%s)\n", key)
+	ctx := ctxArg.(*zedagentContext)
+	trig := cast.CastVifIPTrig(statusArg)
+	if trig.Key() != key {
+		log.Errorf("handleAppVifIPTrigModify key mismatch %s vs %s; ignored\n",
+			key, trig.Key())
+		return
+	}
+	findVifAndTrigAppInfoUpload(ctx, trig.Key())
+}
+
+func handleAppVifIPTrigDelete(ctxArg interface{}, key string,
```

Comment:
Is this a No-Op function? I don't see it deleting anything??

Reply:
don't think we'll have a delete need on this call.